test/acceptance/features/prcalculator.feature

```diff
@@ -1,138 +0,0 @@
-Feature: FFC-PR-CALCULATOR
```

Comment:
Why not modify this file to reflect new changes instead of deleting it?

Reply:
I made a copy of the file to edit and renamed it to reflect the delinked PR calculator updates. The removed file was because it was a duplicate

Reply:
pushing the file, I realise it is missing from the commit